bin/cli.js

```diff
@@ -10,7 +10,15 @@ program.version(version);
 program
   .command('serve [input]')
   .description('starts a development server')
-  .option('-p, --port <port>', 'set the port to serve on. defaults to 1234')
+  .option(
+    '-p, --port <port>',
+    'set the port to serve on. defaults to 1234',
+    parseInt
```

Comment:
Added coercion to the port. Node handles it as-is, but it was resulting in a false error message on Windows:

![image](https://user-images.githubusercontent.com/4730164/34023418-d2cf2dee-e112-11e7-9bef-788c8a45263c.png)